### ISSUE TITLE
Metadata view page: map is not displayed

### ISSUE THREAD

**Issue by etj** [geosolutions-it/iso19139.rndt#160]

**Comment by etj** [comment]
The URL defined here:
```
./web/src/main/webResources/WEB-INF/config-spring-geonetwork.xml:           value="http://ows.terrestris.de/osm/service?SERVICE=WMS&amp;REQUEST=GetMap&amp;VERSION=1.1.0&amp;LAYERS=OSM-WMS&amp;STYLES=default&amp;SRS={srs}&amp;BBOX={minx},{miny},{maxx},{maxy}&amp;WIDTH={width}&amp;HEIGHT={height}&amp;FORMAT=image/png"/>
```
returns a 301 toward the https service.

The default value (and its handling) should have been fixed in later gn versions (we're using a 3.10.4, while 3.10.5 has been released in the meanwhile).
We'll have to upgrade to 3.10.6 anyway to get the changes we pulled into the official core-geonetwork.

**Comment by etj** [comment]
Upgrade gestito nella #162
